src/aura/STG_CMP_CourseConnections/STG_CMP_CourseConnectionsHelper.js

```diff
@@ -30,11 +30,42 @@
     var action = component.get("c.getEnqueueCourseConnectionsBackfill");
     action.setCallback(this, function(response) {
         if(response.getState() === "SUCCESS") {
-          component.set('v.startBackfillMessage', 'Backfill was successfully started.');
+            component.set('v.startBackfillMessage', 'Backfill was started successfully.');
```

Comment:
This message, as well as the error message below, should be a custom label so that it can be translated.